conans/test/functional/workspace/workspace_test.py

```diff
@@ -76,7 +76,7 @@ def package_info(self):
 project(Hello CXX)
 cmake_minimum_required(VERSION 2.8.12)
 include(${{CMAKE_CURRENT_BINARY_DIR}}/conanbuildinfo_multi.cmake)
-conan_basic_setup()
+conan_basic_setup(NO_OUTPUT_DIRS)
```

Comment:
This shouldn't be necessary. ``conan_basic_setup()`` is not setting up output directories, because by default that would be overwriting artifacts.

Making ``conan_basic_setup()`` of ``cmake_multi`` to define OUTPUT_DIRS by default might be breaking.

Reply:
Yes, this is a WIP and the CMakeLists.txt of the test has that to achieve the same behavior